src/main/java/org/apache/mesos/hdfs/util/HDFSConstants.java

```diff
@@ -1,36 +1,39 @@
 package org.apache.mesos.hdfs.util;
 
-public class HDFSConstants {
+/**
+ * defines the constants shared between the scheduler and the executors
+ */
+public interface HDFSConstants {
```

Comment:
Okay, this is fine. Yes, I've seen this before. It is a standard confusion although I find it a bit unclear personally.